src/main/java/com/github/krzysiekjodlowski/ox/App.java

```diff
@@ -6,51 +6,43 @@
 import java.util.Scanner;
 
 /**
+ * At this moment App purpose is to
+ * serve as board generator.
+ * Implementation of this
+ * class will change and should
+ * not be concerned when reviewing.
+ *
  * @author KrzysiekJodlowski
  */
 final class App {
 
     /**
-     * Private according to checkstyle.
+     * Checks if there is any custom input and then creates
+     * game board representation for each case from that file.
+     *
+     * @param args file with custom input (expecting file content as first parameter)
      */
-    private App() {
-
-    }
-
-    /**
-     * In this iteration App checks if there
-     * is any custom input and then creates
-     * game board representation for each case
-     * from that file. Implementation of this
-     * method will change and should not be
-     * concerned when reviewing.
-     * @param args file with custom input
-     * @throws FileNotFoundException when there
-     * is no such file parameter is too big
-     */
-    public static void main(final String[] args)
-            throws FileNotFoundException {
+    public static void main(final String[] args) throws NumberLowerThanOneException {
         Board board;
         int boardSize = 3;
-        BoardPrinter boardPrinter = new BoardPrinter(boardSize);
 
-        if (args.length == 1) {
-            Scanner scanner = new Scanner(
-                    new FileInputStream(new File(args[0]))
-            );
-            while (scanner.hasNextInt()) {
+        if (args.length > 0) {
+            Scanner scanner = null;
+            try {
+                scanner = new Scanner(
+                        new FileInputStream(new File(args[0]))
+                );
+            } catch (FileNotFoundException e) {
+                System.out.println("File not found!");
+            }
+            while (scanner != null && scanner.hasNextInt()) {
                 boardSize = scanner.nextInt();
                 board = new Board(boardSize);
-                boardPrinter = new BoardPrinter(boardSize);
-                System.out.println(
-                        boardPrinter.getBoardRepresentation(board)
-                );
+                System.out.println(board);
```

Comment:
Use some user interface class for printing board.

Reply:
I'll leave this one for now, because as Scanner initialization is done now in try block and I would have to initialize it separately for error message.. It would be awkward and as I said above, main will change soon:)